ensime_shared/client.py

```diff
@@ -101,7 +105,11 @@ def setup_logger():
         self.refactorings = {}
 
         # Queue for messages received from the ensime server.
-        self.queue = Queue()
+#        if sys.version_info >= (3, 5):
+        self.ws_loop = None
```

Comment:
should probably move to a new class, that handles the other thread and exposes thread safe methods

Reply:
added benefit - diff classes can be used for diff python versions